lambda/docker-runner.yaml

```diff
@@ -0,0 +1,111 @@
+# This template depends on having an AWS EFS in place which can be deployed with
+# https://github.com/Sage-Bionetworks/aws-infra/blob/master/templates/EFS/efs.yaml
+AWSTemplateFormatVersion: '2010-09-09'
+Transform: S3Objects
+Description: >-
+  Docker runner product
+Metadata:
+  AWS::CloudFormation::Interface:
+    ParameterGroups:
+      - Label:
+          default: Container Image
+        Parameters:
+          - ImageUri
```

Comment:
A minor point, but a reference to a Docker image is not a URI.  I suggest calling this parameter "ImageReference".  It should allow a reference to a Docker repository image or a commit and takes the form "NAME [:TAG|@DIGEST]".  See https://docs.docker.com/engine/reference/commandline/pull/.  Examples of valid references include:
ubuntu
rocker/r-base
sagebionetworks/some/repo
sagebionetworks/some/repo:1.1
docker.synapse.org/syn123456/my/repo
docker.synapse.org/syn123456/my/repo@sha256:b986e16563abd60ffda0f1ded1e45404ec9296d4b3c7f31bc4f938879f4627d4